packages/enhanced/src/lib/sharing/ConsumeSharedPlugin.ts

```diff
@@ -241,7 +241,7 @@ class ConsumeSharedPlugin {
                   }
                   const { data } = /** @type {DescriptionFile} */ result || {};
                   if (!data) {
-                    if (checkedDescriptionFilePaths) {
+                    if (checkedDescriptionFilePaths?.length) {
```

Comment:
well, i feel stupid

Reply:
It's okay, it's my mistake - I didn't explain it well☺️

Reply:
I'll merge this tomorrow. Then get it synced with rspack and I've already opened the pull request to webpack core as well. Appreciate the contribution. This has definitely been long overdue